src/content/reference/react-dom/components/index.md

```diff
@@ -1,40 +1,38 @@
 ---
-title: "React DOM Components"
+title: "Komponen DOM React"
 ---
 
 <Intro>
-
-React supports all of the browser built-in [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML/Element) and [SVG](https://developer.mozilla.org/en-US/docs/Web/SVG/Element) components.
-
+React mendukung semua komponen [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML/Element) dan [SVG](https://developer.mozilla.org/en-US/docs/Web/SVG/Element) bawaan peramban.
 </Intro>
 
 ---
 
-## Common components {/*common-components*/}
+## Komponen-Komponen Umum {/*common-components*/}
 
-All of the built-in browser components support some props and events.
+Semua komponen bawaan *browser* mendukung beberapa *props* dan *events*.
```

Comment:
Bisa jadi "peramban"

Reply:
Yes according to the Glossary, it's `peramban`.

Reply:
Updated, changed to `peramban`